README.markdown

```diff
@@ -1,14 +1,16 @@
-# The EverCraft Kata
+# The EverCraft Kata [![Dependency Status](https://www.versioneye.com/user/projects/5630a4a536d0ab0016001a17/badge.svg?style=flat)](https://www.versioneye.com/user/projects/5630a4a536d0ab0016001a17)   
```

Comment:
Let's remove the dependency status from here. I like the idea and will implement it via the D&D account.